sw/host/opentitanlib/src/test_utils/lc_transition.rs

```diff
@@ -47,11 +48,10 @@ pub fn trigger_lc_transition(
     reset_delay: Duration,
 ) -> Result<()> {
     // Check the lc_ctrl is initialized and ready to accept a transition request.
-    let lc_status = jtag.read_lc_ctrl_reg(&LcCtrlReg::Status)?;
-    let expected_lc_status =
-        LcCtrlStatusBit::union([LcCtrlStatusBit::Initialized, LcCtrlStatusBit::Ready]);
-    if lc_status != expected_lc_status {
-        return Err(LcTransitionError::LcCtrlNotReady(lc_status).into());
+    let status = jtag.read_lc_ctrl_reg(&LcCtrlReg::Status)?;
+    let status = LcCtrlStatus::from_bits(status).ok_or(LcTransitionError::InvalidState(status))?;
+    if !status.contains(LcCtrlStatus::INITIALIZED | LcCtrlStatus::READY) {
+        return Err(LcTransitionError::LcCtrlNotReady(status).into());
     }
```

Comment:
It looks like this is a behavior change from the prior code. It used to return an error if, for instance, the STATUS register were `INITIALIZED | READY | TOKEN_ERROR`. I think we want a regular equality check here rather than `contains()`.